.github/ISSUE_TEMPLATE/2-bug.yml

```diff
@@ -0,0 +1,29 @@
+name: Bug
+description: Report a bug.
+title: "Bug: <title>"
+labels: [bug, triage]
+body:
+  - type: textarea
+    attributes:
+      label: Bug description
+      description: A concise description of what you're experiencing and what you expected to happen instead.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: Steps to reproduce the behavior.
+      placeholder: |
+        1. In this environment...
+        2. With this config...
+        3. Run '...'
+        4. See error...
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Version
+      description: "Version of the Conduit connector as well as version of the Vitess connector you're using."
```

Comment:
Ummm... Vitess? : )

Reply:
Hah, I renamed it. For some reason, Vitess stuck in my head. Thanks :)

Reply:
Umm... OracleDB?




Just kidding.:D